community/tiff/build

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+./configure \
+	--prefix=/usr \
+	--enable-cxx \
+	--disable-zstd \
+   	--disable-webp \
+    	--without-x
```

Comment:
Are these ^T or spaces? They should be four spaces.

Reply:
Now solved

Reply:
My old eyes are keen enough, it seems!